x-pack/plugins/cases/public/components/case_view/components/assign_users.tsx

```diff
@@ -0,0 +1,223 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback, useEffect, useState } from 'react';
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiButtonIcon,
+  EuiText,
+  EuiLink,
+  EuiPopover,
+  EuiToolTip,
+  EuiLoadingSpinner,
+  EuiSpacer,
+} from '@elastic/eui';
+
+import { UserProfileWithAvatar } from '@kbn/user-profile-components';
+import { useAssignees } from '../../../containers/user_profiles/use_assignees';
+import { CaseAssignees } from '../../../../common/api/cases/assignee';
+import * as i18n from '../translations';
+import { SuggestUsers } from '../../user_profiles/suggest_users';
+import { SidebarTitle } from './sidebar_title';
+import { UserRepresentation } from '../../user_profiles/user_representation';
+import { useCasesContext } from '../../cases_context/use_cases_context';
+import { Assignee } from '../../user_profiles/types';
+
+interface AssigneesListProps {
+  assignees: Assignee[];
+  currentUserProfile?: UserProfileWithAvatar;
+  assignSelf: () => void;
+  togglePopOver: () => void;
+  onAssigneeRemoved: (removedAssigneeUID: string) => void;
+}
+
+const AssigneesList: React.FC<AssigneesListProps> = ({
+  assignees,
+  currentUserProfile,
+  assignSelf,
+  togglePopOver,
+  onAssigneeRemoved,
+}) => {
+  return (
+    <>
+      {assignees.length === 0 ? (
```

Comment:
Would be good to sort this list alphabetically by user name, with the currently logged in user at the top.

<img width="443" alt="image" src="https://user-images.githubusercontent.com/7405507/185197718-a33dd0b6-ea54-483e-8c2a-c335e37043c3.png">

Same with the list in the sidebar:

<img width="295" alt="image" src="https://user-images.githubusercontent.com/7405507/185197918-8718ffde-5a27-4526-a707-2ca9f5c98b55.png">